docs/user-guide/projects/projects-lists/README.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_navigation:
+  title: Projects lists
+  priority: 800
+description: Manage projects lists in OpenProject
+keywords: projects lists, project filters, project overview
+
+---
+
+# Projects lists
+
+In OpenProject you can create projects to collaborate with your team members, track issues, document and share information with stakeholders, organize things. A project is a way to structure and organize your work in OpenProject.
+
+Your projects can be available publicly or internally. OpenProject does not limit the number of projects, neither in the Community edition nor in the Enterprise cloud or in Enterprise on-premises edition.
+
+| Topic                                                       | Content                                                      |
+| ----------------------------------------------------------- | ------------------------------------------------------------ |
+| [Select projects lists](#select-projects-lists)             | Get an overview of all your projects in the projects list.   |
+| [Filter projects lists](#projects-lists-filters)            | Adjust filters in the projects list.                         |
+| [Export project list](#export-projects)                     | You can export the project list to XLS or CSV.               |
+| [Project overarching reports](#project-overarching-reports) | How to create project overarching reports across multiple projects. |
+
+
+## Select projects lists
+
+There are several ways to get an overview of all your projects. You can press the **Projects lists** button at the bottom of the **Select a project** menu in the top left header navigation. 
+
+![projects list button](Projects-list-button.png)
+
+Alternatively, you can use the [**Global modules menu**](../../home/global-modules/#projects) and select *Projects*. You can access that menu either on the left side of the OpenProject application homepage or by using the grid menu icon in the top left corner.
+
+![Select all projects from the global modules menu in OpenProject](view_all_projects_options.png)
+
+You will then get a list of all your projects in OpenProject. 
+
+![A list of all projects in OpenProject](projects-list.png)
+
+### Projects lists view explained
+
+You can use this projects overview to **create a multi project status dashboard** if you include your own [project custom fields](../../system-admin-guide/custom-fields/custom-fields-projects/), e.g. custom status options, Accountable, Project duration, and more.
+
+> **Please note:** Project custom fields are an Enterprise add-on and will only be displayed here for Enterprise on-premises and Enterprise cloud.
```

Comment:
Don't know if that is maintained as Enterprise or not at the end @oliverguenther

Reply:
@oliverguenther please let me know and I can adjust it throughout documentation later on. thanks!